tests/unit/allowable-actions-mixin.spec.ts

```diff
@@ -0,0 +1,239 @@
+import Vue from 'vue'
+import Vuetify from 'vuetify'
+import { shallowMount } from '@vue/test-utils'
+import { getVuexStore } from '@/store'
+import * as FeatureFlags from '@/utils/feature-flags'
+import MixinTester from './mixin-tester.vue'
+
+Vue.use(Vuetify)
+
+const vuetify = new Vuetify({})
+const store = getVuexStore() as any // remove typings for unit tests
+
+// mock global function
+const getFeatureFlag = jest.spyOn(FeatureFlags as any, 'getFeatureFlag')
```

Comment:
Make a note of this! It took me a while to figure out (Google was no help).

This is needed because `getFeatureFlag` is a global function -- not attached to window, global, Vue, etc.

And then you can mock the return value (or implementation)... see below (eg, line 56).